superset-frontend/src/dashboard/components/DeleteComponentButton.tsx

```diff
@@ -36,5 +33,5 @@ export default class DeleteComponentButton extends React.PureComponent {
   }
 }
 
-DeleteComponentButton.propTypes = propTypes;
-DeleteComponentButton.defaultProps = defaultProps;
+export default DeleteComponentButton;
+export { DeleteComponentButtonProps };
```

Comment:
Is there a need to export that?

Reply:
Export the props?
Originally it was, and I thought it would be backward compatible.
You're right. I don't believe it is needed anymore.
I've removed it.